introduction.md

```diff
@@ -1 +1,4 @@
-My name is Nina Denisova. I love git! My favorite CodeSchool path is Javascript (but that's the only one I tried so far).
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?